prelude/__internal__/String.mad

```diff
@@ -0,0 +1,9 @@
+
+show :: a -> String
+export show = (something) => (#- JSON.stringify(something) -#)
+
+
+replace :: String -> String -> String -> String
+export replace = (regex, replacing, input) => (#-
```

Comment:
Maybe not for this PR, but regexes should potentially be a first order component like lists, tuples or records and have their own constructor syntax. For sure the signature of replace will change.